src/common.ts

```diff
@@ -359,6 +365,16 @@ export function defaultErrorRedactor<T = any>(data: {
       if (/^authentication$/.test(key)) {
         headers[key] = REDACT;
       }
+
+      // any casing of `Authorization`
+      if (/^authorization$/.test(key)) {
```

Comment:
For "any casing", shouldn't this be case-insensitive regec?

```suggestion
      if (/^authorization$/i.test(key)) {
```

Reply:
Excellent catch; I've opened a PR to resolve:
- https://github.com/googleapis/gaxios/pull/613